Win10.ps1

```diff
@@ -1659,6 +1693,42 @@ Function HideHiddenFiles {
 	Set-ItemProperty -Path "HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced" -Name "Hidden" -Type DWord -Value 2
 }
 
+# Enable launching folder windows in a separate process
+Function EnableFolderSeparateProcess {
+	Write-Output "Enabling launching folder windows in a separate process..."
+	Set-ItemProperty -Path "HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced" -Name "SeparateProcess" -Type DWord -Value 1
+}
+
+# Disable launching folder windows in a separate process
+Function DisableFolderSeparateProcess {
+	Write-Output "Disabling launching folder windows in a separate process..."
+	Set-ItemProperty -Path "HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced" -Name "SeparateProcess" -Type DWord -Value 0

```

Comment:
Strangely enough, `SeparateProcess` value *does* exist in fresh install, so `Set-ItemProperty` is correct here.